src/molecule/provisioner/ansible.py

```diff
@@ -782,7 +783,7 @@ def manage_inventory(self):
         else:
             self._link_or_update_vars()
 
-    def abs_path(self, path):
+    def abs_path(self, path: str) -> Optional[str]:
```

Comment:
As far as I can tell, this method will always return string. `utils.abs_path` will return `None` only if it receives a falsy argument, but this will not happen here.

Reply:
The only reason why I updated the signature was to keep mypy happy but I am considering removing the method in the future basically because it allowed current bug from being introduced, or at least making it not allow None.